### PR TITLE
LPD-44827 | master

### PR DESCRIPTION
**Pull request by rachaelkoestartyo**
https://liferay.atlassian.net/browse/LPD-44827

### CODE REVIEW

**Comment by rachaelkoestartyo** [comment]
ci:test:sf

**Comment by rachaelkoestartyo** [comment]
ci:test:relevant

File: modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/configuration/AnalyticsConfigurationRegistryImpl.java [hunk 1]
```diff
@@ -235,7 +235,13 @@ private void _disable(long companyId) {
 						AnalyticsDXPEntityBatchExporterConstants.
 							DISPATCH_TRIGGER_NAME_ORDER,
 						AnalyticsDXPEntityBatchExporterConstants.
-							DISPATCH_TRIGGER_NAME_PRODUCT
+							DISPATCH_TRIGGER_NAME_PRODUCT,
```
Comment (marcosapmf) [inline modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/configuration/AnalyticsConfigurationRegistryImpl.java:238]: @rachaelkoestartyo can you sort?

**Review by marcosapmf — CHANGES_REQUESTED** [review]

File: modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/configuration/AnalyticsConfigurationRegistryImpl.java (hunk 1 above)
Comment (rachaelkoestartyo) [reply modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/configuration/AnalyticsConfigurationRegistryImpl.java:238]: i sorted by the class name AnalyticsDXPEntityBatchExporterConstants, AnalyticsMachineLearningConstants. should we sort by the constant name?

Comment (marcosapmf) [reply modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/configuration/AnalyticsConfigurationRegistryImpl.java:238]: Oh I see now :smile:
LGTM
Thanks, Rachael!

**Review by marcosapmf — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
Manually forwarded to https://github.com/brianchandotcom/liferay-portal/pull/157182

**Pull request closed without merging**